website/docs/r/ingest_budget_v2.html.markdown

```diff
@@ -0,0 +1,47 @@
+---
+layout: "sumologic"
+page_title: "SumoLogic: sumologic_ingest_budget_v2"
+description: |-
+  Provides a Sumologic Ingest Budget v2
+---
+
+# sumologic_ingest_budget_v2
+Provides a [Sumologic Ingest Budget v2][1].
```

Comment:
We should explicitly call out the API is in beta and you need to request access to the API.

Reply:
Done